src/dialog/dlgreplacecuecolor.cpp

```diff
@@ -153,11 +153,19 @@ DlgReplaceCueColor::DlgReplaceCueColor(
 
     // Update dialog widgets when conditions checkboxes are (un)checked
     connect(checkBoxCurrentColorCondition,
+#if QT_VERSION >= QT_VERSION_CHECK(6, 7, 0)
+            &QCheckBox::checkStateChanged,
+#else
             &QCheckBox::stateChanged,
+#endif
```

Comment:
ideally the slot signature would also change accordingly to take the enum instead of the `int`...